### PR TITLE
Export answers

### PR DESCRIPTION
**Pull request by Taces**
Added a button for speakers to download the answers as a csv file. The button is located in the panels where the answers can be looked at. Excluded Hotspot-Questions.

### CODE REVIEW

**Comment by kqc-real** [comment]
:+1: Again, this PR looks great! Something several lecturers have been waiting for a long time.

File: src/main/webapp/app/view/FreetextAnswerPanel.js [hunk 1]
```diff
@@ -124,7 +124,32 @@ Ext.define('ARSnova.view.FreetextAnswerPanel', {
 			docked: 'top',
 			ui: 'light',
 			title: Ext.util.Format.htmlEncode(this.questionObj.subject),
-			items: [this.backButton]
+			items: [this.backButton, {
```
Comment (tekay) [inline src/main/webapp/app/view/FreetextAnswerPanel.js]: It's best to define a var for the export button (like the backButton).

Besides, i'd suggest relocating the code for the export into the QuestionExport Controller for a cleaner view-code.

File: src/main/webapp/app/view/speaker/QuestionStatisticChart.js [hunk 2]
```diff
@@ -165,7 +164,36 @@ Ext.define('ARSnova.view.speaker.QuestionStatisticChart', {
 				handler: this.toggleCorrectHandler,
 				hidden: !hasCorrectAnswers() || this.questionObj.questionType === 'grid' ||
 					(ARSnova.app.userRole === ARSnova.app.USER_ROLE_STUDENT && !this.questionObj.showAnswer)
-			}]
+			}, {
+				xtype: 'button',
+				text: Messages.QUESTIONS_CSV_EXPORT_ANSWERS_BUTTON,
+				align: 'right',
+				handler: function () {
```
Comment (tekay) [inline src/main/webapp/app/view/speaker/QuestionStatisticChart.js]: When the export function is in the QuestionExport Controller, you can reuse it here to not repeat yourself.

**Comment by Taces** [comment]
I now put the export button into it's own var and moved the download code into the QuestionExport controller, didn't really know about that beforehand as I'm not that well into your codebase.
Also merged the changes you made as there was now a conflict in QuestionExport controller.

**Pull request merged**